lib/main.dart

```diff
@@ -3,235 +3,159 @@ import 'dart:async';
 import 'package:animate_do/animate_do.dart';
 import 'package:flutter/material.dart';
 import 'package:http/http.dart' as http;
+import 'package:proyectos_flutter/avatar.dart';
+import 'package:proyectos_flutter/icons/like_icon.dart';
+import 'package:proyectos_flutter/icons/linkedin_icon.dart';
+import 'package:proyectos_flutter/icons/mail_icon.dart';
+import 'package:proyectos_flutter/icons/phone_icon.dart';
+import 'package:proyectos_flutter/styles.dart';
 
-void main() => runApp(MyApp());
+import 'personal_info.dart';
+
+void main() => runApp(const MyApp());
 
 class MyApp extends StatefulWidget {
+  const MyApp({Key? key}) : super(key: key);
+
   @override
   _BusinessCard createState() => _BusinessCard();
 }
-class PersonalInfo {
-
-  final String name;
-  final String position;
-  final String email;
-  final String phoneNumber;
-  final String linkedin;
-
-  PersonalInfo({
-    required this.name,
-    required this.position,
-    required this.email,
-    required this.phoneNumber,
-    required this.linkedin,
-  });
-
-  factory PersonalInfo.fromJson(Map<String, dynamic> json) {
-    return PersonalInfo(
-      name: json['name'],
-      position: json['position'],
-      email: json['email'],
-      phoneNumber: json['phonenumber'],
-      linkedin: json['linkedin'],
-    );
-  }
 
-}
 class _BusinessCard extends State<MyApp> {
-int _counter = 0;
-late Future<PersonalInfo> futurePersonalInfo;
-
-@override
-void initState() {
-  super.initState();
-  futurePersonalInfo = fetchPersonalInfo();
-}
+  int _counter = 0;
+  late Future<PersonalInfo> futurePersonalInfo;
 
-void incrementCounter(){
-  setState(() {
-    _counter++;
-  });
-}
+  @override
+  void initState() {
+    super.initState();
+    futurePersonalInfo = fetchPersonalInfo();
+  }
 
+  void incrementCounter() {
+    setState(() {
+      _counter++;
```

Comment:
remember to use meaningful names, _counter ? what is this counting ? votes, likes, purchases, products, if I just see counter, I do not know what is this variable counting, neither the function name right ?